### DISCUSSION TITLE
Missing type `InputMaybe<T>` in generated gqlTypes/globalTypes.ts file

### DISCUSSION THREAD

**Discussion by tilman** [dotansimha/graphql-code-generator#7317] [Q&A] [answered]
Hey, I'm using graphql-codegen together with following plugins
      - "typescript"
      - "typescript-operations"
      - "typescript-graphql-request"

Right now I have the Problem, that one type is missing which get's referenced in other types quite often. The missing type is `InputMaybe<T>` which should be something like this:
```
export type InputMaybe<T> = Maybe<T>;
```
Any Idea why this type is missing or how I can extend the globalTypes.ts with this type manually?

**Comment by tilman** [comment, marked as answer]
Seems like I solved my issue. I was coping the content of `.graphqlrc.yml` into `codegen.yml` and executed it instead with `graphql-codegen -c codegen.yml` and now everything works. So It seems like there is a bug in parsing of `.graphqlrc.yml` files?

**Reply by dotansimha** [reply]
Not sure if that's the actual issue - what version of the plugins are you using?
